### ISSUE TITLE
accessibility issue - 24 (medium)- wrap with dl in compact_index_default.html.erb

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#243]
The description list items on the search results page do not have parent <dl> elements. Definition list items (dt and/or dd) must be wrapped in parent dl elements to be valid. This enables screen reader users to understand the proper hierarchy of information in the list.

**Comment by yulgit1** [comment]
resolved:https://github.com/ycba-cia/blacklight-collections2/commit/c3df5496167b29fa7ba9273146c11ab09e1e0421
